### DISCUSSION TITLE
System.IO.FileLoadException: 'Could not load file or assembly 'Aspects.Universal, Version=1.0.1.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)'

### DISCUSSION THREAD

**Discussion by emialmasia** [pamidur/aspect-injector#224] [General]
System.IO.FileLoadException: 'Could not load file or assembly 'Aspects.Universal, Version=1.0.1.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)'

**Comment by emialmasia** [comment]
.Net Framework 4.6.2

**Comment by emialmasia** [comment]
Is there anyone here?

**Comment by emialmasia** [comment]
Referenced assembly 'Aspects.Universal, Version=1.0.1.0, Culture=neutral, PublicKeyToken=null' does not have a strong name.

In Debug

**Comment by emialmasia** [comment]
The only way I can make it work
It was downloaded and installed from nuget

"Brutal.Dev.StrongNameSigner"

Thanks for the help

**Comment by pamidur** [comment]
Could you please try https://www.nuget.org/packages/Aspects.Universal/1.0.2 and see if it helps?
